pkg/gui/controllers/helpers/merge_and_rebase_helper.go

```diff
@@ -201,22 +201,30 @@ func (self *MergeAndRebaseHelper) RebaseOntoRef(ref string) error {
 	if ref == checkedOutBranch {
 		return self.c.ErrorMsg(self.c.Tr.CantRebaseOntoSelf)
 	}
-	prompt := utils.ResolvePlaceholderString(
-		self.c.Tr.ConfirmRebase,
-		map[string]string{
-			"checkedOutBranch": checkedOutBranch,
-			"selectedBranch":   ref,
+	menuItems := []*types.MenuItem{
```

Comment:
We still want to tell the user that we're rebasing from `checkedOutBranch` to `ref`. So I'm thinking:
```
Title: Rebase '{{checkedOutBranch}}' onto '{{ref}}'
Item 1: s: Simple rebase
Item 2: i: Interactive rebase
```

And I'd add a tooltip for item 2 saying 'Begin an interactive rebase with a break at the start, so you can update the TODO commits before continuing'